src/roll20/content-script.js

```diff
@@ -253,6 +253,8 @@ function rollSkill(request, custom_roll_dice = "") {
         let d20 = request.reliableTalent ? "{1d20, 0d0 + 10}kh1" : "1d20";
         if (request.silverTongue && (request.skill === "Deception" || request.skill === "Persuasion"))
             d20 = "{1d20, 0d0 + 10}kh1"
+        if (request.indomitableMight && request.skill === "Athletics")
+            d20 = `{1d20, 0d0 + ${(request.character.abilities[0][2])-parseInt(request.modifier)}}kh1`;
```

Comment:
This actually wouldn't work well for those using a custom dice in the modifier

Reply:
Man this one sucks... Any further ideas here?

Reply:
This issue (not this specific code) affects rolls on DnDBeyond and FoundryVTT too... It's the issue of the keep-highest (or min) not knowing the custom-die result to work things properly first... If you have any ideas, that's great, otherwise I might just drop this...